web_scraper/test_script.py

```diff
@@ -0,0 +1,15 @@
+import scrapy
+from scrapy.crawler import CrawlerRunner
+## crochet plays with Twisted's Reactors to avoid ReactorNotRestartable error
+from crochet import setup 
+setup()
+
+def run_spider(spiderClass):
+    crawler = CrawlerRunner()
+    crawler.crawl(spiderClass)
+
+######################## How to run #####################
```

Comment:
@junpuf @saimidu This is a test script to show how one could call the crawler from within the script.

Reply:
great! this will allow us to use existing testing workflow for this as well.